src/request_queues.js

```diff
@@ -227,24 +227,31 @@ export default {
      *                                        and to the end of the queue otherwise.
      * @param {String} [options.token] - Your API token at apify.com. This parameter is required
      *                                   only when using "username~queue-name" format for queueId.
+     * @param {String} [options.clientId] - Unique ID identifying client accessing the request queue.
```

Comment:
As mentioned in PR on BitBucket, I'd strongly prefer to call this `clientKey`, it's not really an ID.